regression/cbmc-incr-smt2/nondeterministic-int-assert/nondet_symbol.c

```diff
@@ -0,0 +1,11 @@
+extern int john();
```

Comment:
This.. doesn't have any associated `.desc` files?

I doubt this is being run as a test atm.